src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -561,14 +568,14 @@ const char* COBJMeshFileLoader::readTextures(const SContext& _ctx, const char* b
 	}
 	if (clamp)
     {
-        for (size_t i=0; i<_IRR_MATERIAL_MAX_TEXTURES_; i++)
+        for (size_t i=0; i< video::OGL_STATE_MAX_TEXTURES; i++)
         {
-            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapU = video::ETC_CLAMP_TO_EDGE;
-            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapV = video::ETC_CLAMP_TO_EDGE;
-            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapW = video::ETC_CLAMP_TO_EDGE;
+            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapU = ISampler::E_TEXTURE_CLAMP::ETC_CLAMP_TO_EDGE;
+            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapV = ISampler::E_TEXTURE_CLAMP::ETC_CLAMP_TO_EDGE;
+            currMaterial->Material.TextureLayer[i].SamplingParams.TextureWrapW = ISampler::E_TEXTURE_CLAMP::ETC_CLAMP_TO_EDGE;
         }
     }
-	for (size_t i = 0; i < _IRR_MATERIAL_MAX_TEXTURES_; i++)
+	for (size_t i = 0; i < video::OGL_STATE_MAX_TEXTURES; i++)
 		currMaterial->Material.TextureLayer[i].SamplingParams.AnisotropicFilter = 16u;
```

Comment:
you need to create sampler objects now instead, in the CDescriptorSetLoaderMTL

And make sure you cache them in the `builtin` path (if caching flags allow)